composer.json

```diff
@@ -24,8 +24,8 @@
         "guzzle/guzzle": "3.*",
         "imagine/imagine": "~0.6.1",
         "liip/theme-bundle": "1.3.*",
-        "massive/search-bundle": "0.13.*",
-        "sulu/document-manager": "0.6.*",
+        "massive/search-bundle": "dev-develop",
```

Comment:
No, but should it not be on `dev-develop` in anycase when merging into `dev-develop` ?